custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -197,6 +198,25 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
         ],
     };
 
+    if (haveAccessToFeatures) {
+        var numberOfCbeConducted = {
+            'mData': 'num_awcs_conducted_cbe',
+            'heading': 'Number of CBE Conducted',
+            'tooltipValue': 'Number of Community-Based Events (CBEs) organised by an AWC in a month',
+            'columnValueType': 'raw',
+            'columnValueIndicator': 'num_awcs_conducted_cbe',
```

Comment:
Hey, if you want the number of CBE/VHND itself, the columns to use are `cbe_conducted, vhnd_conducted,`.   `num_awcs_conducted_cbe` is the number of AWCS conducted at least 2 CBE. and `num_awcs_conducted_vhnd` is number of awcs conducted at least 1 CBE.

Reply:
I agree. But the definition that you mentioned is for all the levels above sector right? In sector level, I think `num_awcs_conducted_cbe` returns cbe conducted by individual awc in the sector.

Reply:
These two columns are being added only at the sector level

Reply:
which tooltip you refered? I just checked it says `if awcs has conducted at least 2 Cbes then yes else no`.
These columns are the actual count of CBE and VHND yeah?
if you see `num_awcs_conducted_cbe` is calculated here as if the CBE count is>1 then its counted as 1 [here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py#L100). The actual count is stored [here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py#L101)